tests/Kiota.Builder.Tests/Refiners/PythonLanguageRefinerTests.cs

```diff
@@ -369,7 +429,6 @@ public async Task CorrectsCoreType()
         Assert.Empty(model.Properties.Where(x => PathParametersDefaultValue.Equals(x.DefaultValue)));
         Assert.Empty(model.Methods.Where(x => DeserializeDefaultName.Equals(x.ReturnType.Name)));
         Assert.Empty(model.Methods.SelectMany(x => x.Parameters).Where(x => serializerDefaultName.Equals(x.Type.Name)));
-        Assert.Single(constructorMethod.Parameters.Where(x => x.Type is CodeComposedTypeBase));
```

Comment:
Maybe bring this back and validate the current type to assert the change is intentional?